lib/src/isolate_manager/isolate_manager_web.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:async';
+
+import 'package:dash_kit_network/dash_kit_network.dart';
+import 'package:dash_kit_network/src/isolate_manager/isolate_manager.dart';
+
+//  The IsolateManager class implements the IsolateManagerInterface and provides
+//  an interface for managing isolates in the DashKit network library.
+// ignore: prefer-match-file-name
+class IsolateManager extends IsolateManagerInterface {
+  IsolateStatus _isolateStatus = IsolateStatus.created;
+
+  @override
+  Future<void> start() async {
+    assert(_isolateStatus == IsolateStatus.created, 'Isolate already started');
```

Comment:
let's move asserts to the abstract class maybe?

and add an assert to the `send` method as well that we are in `initialized` state

Reply:
Moved it.